### PR TITLE
GitHub Actions dev:check ci

### PR DESCRIPTION
**Pull request by crazy-max**
Follow-up #12353 

Web check failed without any log:

```
Running web check... 
failed (0.62s)
```

Git error right after `php artisan dev:check -vv ci`, don't know why:

```
fatal: ambiguous argument 'master': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
Running PHP lint check... success (20.91s)
Running Python lint check... success (5.82s)
...
```

### CODE REVIEW

**Comment by Jellyfrog** [comment]
It fails because the branch master is not available. Guessing you have to change it to origin/master

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+      - 'doc/**'
+      - 'mkdocs.yml'
+
+env:
+  APP_ENV: testing
+  DB_NAME: db
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
```
Comment (Jellyfrog) [inline .github/workflows/test.yml:28]: We want to be able to use different version here. 
Check https://github.com/librenms/librenms/pull/12035

Tldr is that we want to test different net-snmp and mysql versions

Comment (crazy-max) [reply .github/workflows/test.yml:28]: If you just want to test other releases of mysql you just have to change the version of the mysql Docker service (or create a new one with your desired release). This way we are sure of the current MySQL release we want to use: https://github.com/librenms/librenms/pull/12392/files#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88R46

Comment (f0o) [reply .github/workflows/test.yml:28]: I agree with SQL versions supplied via the services tag.

However net-snmp version changes shouldnt really matter much although we should keep track of the LTS versions of ubuntu just to be sure that compatibility is ensured.

Comment (murrant) [reply .github/workflows/test.yml:28]: I'm fine with using a new version, just aware that some test data will fail.

**Comment by murrant** [comment]
If you can supply a list of changed files in the `FILES` env variable, the dev:check command will not need to run the git command to find the differences.

**Review by murrant — APPROVED** [review]
LGTM, removed the extra test units run, added a couple visual things.

**Comment by murrant** [comment]
@crazy-max @Jellyfrog thanks for getting this going :D
